app/server.rb

```diff
@@ -1,6 +1,14 @@
 require 'sinatra'
 require 'aws-record'
 
+before do
```

Comment:
this will ensure that `params` corespond with AWS API Gateway proxy set in lambda.rb `ENV['rack.input]`